test/unit/run.test.js

```diff
@@ -70,4 +71,58 @@ describe('run module', function () {
             }
         ], done);
     });
+
+    it('must correctly resolve conflicts between iterationData.length and iterationCount', function (done) {
```

Comment:
Okay, how do we verify the number of iterations that occurred from within an `exec` call?